test/gleam/result_test.gleam

```diff
@@ -196,6 +196,34 @@ pub fn all_test() {
   |> should.equal(Error("a"))
 }
 
+pub fn any_test() {
+  [Error(1), Ok(2), Ok(3)]
+  |> result.any
+  |> should.equal(Ok(2))
+
+  [Ok(1), Error("a"), Error("b"), Ok(3)]
+  |> result.any
+  |> should.equal(Ok(1))
+
+  [Error("a"), Error("b"), Error("c")]
+  |> result.any
+  |> should.equal(Error(["a", "b", "c"]))
+}
```

Comment:
Maybe you could add a test that takes into account the empty list as input. In that case is the expected result an `Error` wrapping an empty list?